src/ape/api/providers.py

```diff
@@ -752,6 +752,10 @@ def _sanitize_web3_url(msg: str) -> str:
     parts = msg.split("URI: ")
     prefix = parts[0].strip()
     rest = parts[1].split(" ")
+
+    # * To remote the `,` from the url http://127.0.0.1:8545,
+    if "," in rest[0]:
+        rest[0] = rest[0].strip(",")
```

Comment:
```suggestion
        rest[0] = rest[0].rstrip(",")
```

since it is only on the right side?

Reply:
Yes I'm not seeing the error after running in debug mode

Reply:
yah